nucypher/config/base.py

```diff
@@ -304,7 +304,7 @@ class CharacterConfiguration(BaseConfiguration):
     'Sideways Engagement' of Character classes; a reflection of input parameters.
     """
 
-    VERSION = 5  # bump when static payload scheme changes
+    VERSION = 6  # bump when static payload scheme changes
```

Comment:
Our current publicly released configuration version is 4 - https://github.com/nucypher/nucypher/blob/main/nucypher/config/base.py#L325C59-L325C59.
So:
1. Do we need to go from 5->6 or can we just stay at 5? 5 was only an internal unreleased dev change (https://github.com/nucypher/nucypher/pull/3043). Looks like we already have a script for v4 -> v5 (https://github.com/nucypher/nucypher/commit/2d0f02a287277171bb68c508d8cb43e5f6b72f06). Can that be piggy-backed upon?
2. Do we need a related migration script for Ursula as part of this change? Whatever version we end up using (5 or 6), we'll need a migration script from v4.

Reply:
I don't mind bumping the version here again as a safety measure, but I do agree that a migration script may be useful.  I'll add one.

Reply:
✔️